source/windows/1_installation.md

```diff
@@ -1,76 +1,85 @@
 
 ![Mongo Defaults](./pics/defaultSetupMongo.png)
 
-
-The fourth dialog box will ask if Mongo Compass should be installed. Feel free to choose what is best for your situation, however for the purposes of Teclado courses, this isn't needed.  Uncheck it, and click next. 
+The fourth dialog box will ask if Mongo Compass should be installed. Feel free to choose what is best for your situation, however for the purposes of Teclado courses, this isn't needed.  Uncheck it, and click **next**. 
 
 ![Mongo Compass](./pics/uncheckCompas.png)
 
-Finally, click install
+Finally, click **install**.
+
+![Mongo Install](./pics/mongorIinstall.png)
 
-![Mongo Install](./pics/mongoinstall.png)
+And click **finish**.
 
-And click finish.
 ![Mongo Finished](./pics/mongoFinish.png)
 
 ## Setting up Windows
+
 If you open CMD in windows and type in mongo, you will see that mongo doesn't work yet.  This is because the installer doesn't tell Windows where mongo lives in Windows. 
 
 ![Mongo not a recognized](./pics/mongoNotWorking.png)
 
-To fix this, using the Window search, look for **environmental variables**
+To fix this, using the Windows search look for **environment variables**.
 
-![Searching for Environmental Variables](./pics/searchForEnvironmental.PNG)
+![Searching for Environment Variables](./pics/searchForEnvironmental.PNG)
 
-This will open up edit Environmental Variable's properties box.  click once on the **path** property, and then click **edit**.
+This will open up edit Environment Variable's properties box.  Click once on the **path** property, and then click **edit**.
 
-![Environmental Variable](./pics/environmentalvariablesEdit.png)
+![Environment Variable](./pics/environmentalvariablesEdit.png)
 
-This will bring up the Edit Environmental Variables dialog box.  Click **New**
+This will bring up the Edit Environment Variables dialog box.  Click **New**.
 
 ![Clicking New](./pics/CreateNewEnvironmentalVariable.png)
 
 Once new is clicked, a cursor will show up in a new editable row. There you should paste the following:
 
-`C:\Program Files\MongoDB\Server\4.0\bin`
+```
+C:\Program Files\MongoDB\Server\4.0\bin
+```
 
-![New Environmental Variable](./pics/pasteMongoLocationIn.png)
+![New Environment Variable](./pics/pasteMongoLocationIn.png)
 
-Click Ok, and ok again until all dialog boxes are closed. Then run cmd once more and attempt to execute. Mongo. 
+Click Ok, and ok again until all dialog boxes are closed. Then run cmd.exe once more (closing and re-opening if necessary) and attempt to execute `mongo`. 
 
 ![Working Mongo](./pics/workingMongo.PNG)
 
-Note: In the course videos, you will see that I run mongod in the terminal.  This is not needed in Windows, as it is installed as a service. 
-
+::: tip
+In the course videos you will see that I run `mongod` in the terminal.  This is not needed in Windows, as it is installed as a service. 
+:::
```

Comment:
This `::: tip` is a Vuepress custom block (https://vuepress.vuejs.org/guide/markdown.html#custom-containers)

Reply:
I went ahead and implimented this on ryan-dev branch.